search-commons/src/main/java/no/unit/nva/search2/resource/ResourceClient.java

```diff
@@ -7,19 +7,28 @@
 import static nva.commons.core.attempt.Try.attempt;
 import java.net.http.HttpClient;
 import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.function.BinaryOperator;
 
 import no.unit.nva.search2.common.jwt.CachedJwtProvider;
 import no.unit.nva.search2.common.OpenSearchClient;
 import no.unit.nva.search2.common.records.SwsResponse;
+import no.unit.nva.search2.common.records.SwsResponse.SwsResponseBuilder;
 import nva.commons.core.JacocoGenerated;
 import nva.commons.secrets.SecretsReader;
 
+
 /**
  * @author Stig Norland
  */
 public class ResourceClient extends OpenSearchClient<SwsResponse, ResourceSearchQuery> {
 
     private final UserSettingsClient userSettingsClient;
+    @SuppressWarnings("PMD.DoNotUseThreads")
+    private static final ExecutorService executorService = Executors.newFixedThreadPool(2);
```

Comment:
This might give rise to problem in aws lambda thread pool...

Reply:
It should be OK, but I believe it is curiously dependent on the amount of memory the Lambda is assigned